etc/skel/.config/hypr/config/keybinds.conf

```diff
@@ -40,53 +40,53 @@ bindd = , XF86AudioPrev, Previous track, exec, playerctl previous
 
 # ======= Screen Brightness =======
 
-bindde = , XF86MonBrightnessUp, Increases brightness 5%, exec, brightnessctl s +5%
-bindde = , XF86MonBrightnessDown, Decreases brightness 5%, exec, brightnessctl s 5%-
+bindd = , XF86MonBrightnessUp, Increases brightness 5%, exec, brightnessctl s +5%
```

Comment:
One last change, can you change these from `bindd` to `bindel` which would allow them to be repeated when held + usable even on a lockscreen.

Quote from Wayland docs:

> Clients can use this interface to prevent input events from being sent to any surfaces but its own, which is useful for example in lock screen software. It is assumed that access to this interface will be locked down to whitelisted clients by the compositor.

Reply:
And this one from Hyprland wiki:

> l -> locked, will also work when an input inhibitor (e.g. a lockscreen) is active.

Reply:
Done, I did remove the description though since it probably wont work that way.